modules/aerodyn/src/AeroDyn_Types.f90

```diff
@@ -358,7 +358,7 @@ MODULE AeroDyn_Types
 ! =======================
 ! =========  AD_MiscVarType  =======
   TYPE, PUBLIC :: AD_MiscVarType
-    TYPE(RotMiscVarType) , DIMENSION(:), ALLOCATABLE  :: rotors      !< MiscVars for each rotor [-]
+    TYPE(RotMiscVarType) , DIMENSION(:), ALLOCATABLE  :: rotors      !< - [MiscVars for each rotor]
```

Comment:
Extra dash (from extra dash in AeroDyn_Registry.txt

Reply:
Resolved in #1596